libspdm dosent compile on aarch64 - both clang or gcc

ARCH = aarch64
TOOLCHAIN = GCC
GCOV = OFF
TARGET = Release
CRYPTO = openssl
ENABLE_BINARY_BUILD=0; Building openssl library from source

```
Scanning dependencies of target spdm_common_lib
[  0%] Building C object os_stub/cryptlib_openssl/CMakeFiles/cryptlib_openssl.dir/cipher/aead_aes_gcm.c.o
Scanning dependencies of target spdm_requester_lib
[  0%] Building C object os_stub/cryptlib_openssl/CMakeFiles/cryptlib_openssl.dir/cipher/aead_chacha20_poly1305.c.o
Scanning dependencies of target spdm_responder_lib
In file included from /home/olympus/alok/libspdm/include/hal/base.h:13,
                 from /home/olympus/alok/libspdm/os_stub/cryptlib_openssl/internal_crypt_lib.h:14,
                 from /home/olympus/alok/libspdm/os_stub/cryptlib_openssl/cipher/aead_aes_gcm.c:14:
/home/olympus/alok/libspdm/include/hal/aarch64/processor_bind.h:7: error: unterminated #ifndef
    7 | #ifndef __PROCESSOR_BIND_H__
      |
In file included from /home/olympus/alok/libspdm/os_stub/cryptlib_openssl/internal_crypt_lib.h:14,
                 from /home/olympus/alok/libspdm/os_stub/cryptlib_openssl/cipher/aead_aes_gcm.c:14:
/home/olympus/alok/libspdm/include/hal/base.h:589:9: error: unknown type name ‘uintn’
  589 | typedef uintn *BASE_LIST;
      |         ^~~~~
/home/olympus/alok/libspdm/include/hal/base.h:663:12: error: ‘boolean’ undeclared here (not in a function)
  663 |     sizeof(boolean) == 1,
      |            ^~~~~~~
/home/olympus/alok/libspdm/include/hal/base.h:663:5: error: expression in static assertion is not an integer
  663 |     sizeof(boolean) == 1,
      |     ^~~~~~
/home/olympus/alok/libspdm/include/hal/base.h:666:12: error: ‘int8_t’ undeclared here (not in a function)
  666 |     sizeof(int8_t) == 1,
      |            ^~~~~~
/home/olympus/alok/libspdm/include/hal/base.h:666:5: error: expression in static assertion is not an integer
  666 |     sizeof(int8_t) == 1,
      |     ^~~~~~
[  0%] Building C object library/spdm_common_lib/CMakeFiles/spdm_common_lib.dir/libspdm_com_context_data.c.o
/home/olympus/alok/libspdm/include/hal/base.h:669:12: error: ‘uint8_t’ undeclared here (not in a function)
  669 |     sizeof(uint8_t) == 1,
      |            ^~~~~~~
/home/olympus/alok/libspdm/include/hal/base.h:669:5: error: expression in static assertion is not an integer
  669 |     sizeof(uint8_t) == 1,
```

--- a/include/hal/aarch64/processor_bind.h
+++ b/include/hal/aarch64/processor_bind.h
@@ -66,6 +66,8 @@
 
 #pragma warning(disable : 4703)
 
+#endif
+
 #ifndef LIBSPDM_STDINT_ALT
 #include <stdint.h>
 #else

